### ISSUE TITLE
npm start on example gives "Error: Cannot find module 'feathers-hooks-common/lib/populate'"

### ISSUE THREAD

**Issue by IBwWG** [eddyystop/feathers-reduxify-services#3]
Bump version in devDependencies in package.json to 2.0.3 to fix (populate first committed then, it looks like.)

**Comment by IBwWG** [comment]
Also the `webpack` dependency is undeclared.

**Comment by dancancro** [comment]
I got this one too:
```
// ♥ npm run build

> users-auth@0.0.1 prebuild /Users/Dan/work/bernierebuttals/feathers-reduxify-services/example
> rm -rf ./public/dist


> users-auth@0.0.1 build /Users/Dan/work/feathers-reduxify-services/example
> webpack --progress --colors

Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.resolve.extensions[0] should not be empty.
```

**Comment by eddyystop** [comment]
Since the webpack1 dependency is missing, webpack2 can run and it doesn't like the null string in ` extensions: ['', '.js', '.jsx'],`.

The hooks should be brought up to the Auk release, i.e. all hooks are exposed directly under `feathers-hooks-common`.

This affects the example, not the feature itself. I'll address this shortly.

**Comment by eddyystop** [comment]
Released v1.0.0.

Repo has been updated to Feathers Auk release. All dependencies are now current. Test and build now work on Windows.
